XRTK-Core/Assets/Assets.index

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
What does this file do?

Reply:
This file is created by the `Quick Search` upm package. I accidentally checked it in.